podcasts/podcasts-Info.plist

```diff
@@ -181,6 +144,31 @@
 					<string>AppIcon-Dark40x40@2x~ipad</string>
 				</array>
 			</dict>
+			<key>AppIcon-Default</key>
```

Comment:
Was these changes made by Xcode?

Reply:
Yeah, it looks like a bunch of changes happened when I added the new type identifier and document type. I believe they are mostly position changes in the file.